src/libsyntax_pos/symbol.rs

```diff
@@ -510,10 +515,19 @@ impl Interner {
         symbol.0.as_usize() >= self.strings.len()
     }
 
-    pub fn get(&self, symbol: Symbol) -> &str {
+    /// Get the chars of a normal (non-gensym) symbol.
```

Comment:
The distinction between gensymed and non-gensymed should never leak into a public interface.
For the `Symbol` users gensyms should look just like usual names (that happen to have some unique hashes appended).
`is_gensymed` is unnecessary after https://github.com/rust-lang/rust/pull/51492 and every its use is a mistake.

Reply:
Gensyms are not a different "kind" of symbols.

Reply:
Ok, I will fix.